### PR TITLE
Linked names

### PR DESCRIPTION
**Pull request by EgbertRijke**

### CODE REVIEW

File: CONTRIBUTORS.toml [hunk 1]
```diff
@@ -75,16 +75,19 @@ github = "EleonoreMangel"
 displayName = "Bryan Lu"
```
Comment (fredrik-bakke) [inline CONTRIBUTORS.toml:75]: I just noticed, the spelling of Bryan's name in `100-theorems` is wrong. Would you mind correcting it?

Comment (fredrik-bakke) [reply CONTRIBUTORS.toml:75]: Oh, you did

File: src/elementary-number-theory/bezouts-lemma-integers.lagda.md [hunk 2]
```diff
@@ -38,9 +38,11 @@ open import foundation.unit-type
 
 </details>
 
-Bézout's Lemma is the 60th theorem on Freek Wiedijk's list of
+Bézout's Lemma is the 60th theorem on
+[Freek Wiedijk's](http://www.cs.ru.nl/F.Wiedijk/) list of
 [100 theorems](literature.100-theorems.md) {{#cite 100theorems}}. It was
-originally added to agda-unimath by [Brian Lu](https://blu-bird.github.io).
+originally added to agda-unimath by
+[[Bryan Lu](https://blu-bird.github.io)](https://blu-bird.github.io).
```
Comment (fredrik-bakke) [inline src/elementary-number-theory/bezouts-lemma-integers.lagda.md:45]:
```suggestion
[Bryan Lu](https://blu-bird.github.io).
```

File: src/elementary-number-theory/bezouts-lemma-natural-numbers.lagda.md [hunk 3]
```diff
@@ -61,9 +61,11 @@ predicate `P : ℕ → UU lzero` given by
 is decidable, because `P z ⇔ [y]_x | [z]_x` in `ℤ/x`. The least positive `z`
 such that `P z` holds is `gcd x y`.
 
-Bézout's Lemma is the 60th theorem on Freek Wiedijk's list of
+Bézout's Lemma is the 60th theorem on
+[Freek Wiedijk's](http://www.cs.ru.nl/F.Wiedijk/) list of
 [100 theorems](literature.100-theorems.md) {{#cite 100theorems}}. It was
-originally added to agda-unimath by [Brian Lu](https://blu-bird.github.io).
+originally added to agda-unimath by
+[[Bryan Lu](https://blu-bird.github.io)](https://blu-bird.github.io).
```
Comment (fredrik-bakke) [inline src/elementary-number-theory/bezouts-lemma-natural-numbers.lagda.md:68]:
```suggestion
[Bryan Lu](https://blu-bird.github.io).
```

File: src/literature/100-theorems.lagda.md [hunk 4]
```diff
@@ -12,7 +13,7 @@ module literature.100-theorems where
 
 ### [3. The Denumerability of the Rational Numbers](https://www.cs.ru.nl/~freek/100/#3) {#3}
 
-Author: Fredrik Bakke
+Author: [Fredrik Bakke](https://www.ntnu.edu/employees/fredrik.bakke)
```
Comment (fredrik-bakke) [inline src/literature/100-theorems.lagda.md:16]: Can we make the "Author:" font bold? I find it looks a bit unorganized in the default font.

Comment (EgbertRijke) [reply src/literature/100-theorems.lagda.md:16]: Yep

**Review by fredrik-bakke — APPROVED** [review]
Nice! I'm hoping to have a better home page at some point, but my employer's page works for now.

**Comment by EgbertRijke** [comment]
Thanks for catching the errors! It should be good now

**Pull request merged**